lib/ExpensiMark.js

```diff
@@ -547,15 +547,18 @@ export default class ExpensiMark {
             },
             {
                 name: 'userMention',
-                regex: /<mention-user accountID="(\d+)" *\/>/gi,
-                replacement: (match, g1, offset, string, extras) => {
-                    const accountToNameMap = extras.accountIdToName;
-                    if (!accountToNameMap || !accountToNameMap[g1]) {
-                        Log.alert('[ExpensiMark] Missing account name', {accountID: g1});
-                        return '@Hidden';
-                    }
+                regex: /(?:<mention-user accountID="(\d+)" *\/>)|(?:<mention-user>(.*?)<\/mention-user>)/gi,
+                replacement: (match, g1, g2, offset, string, extras) => {
+                    if (g1) {
+                        const accountToNameMap = extras.accountIDToName;
+                        if (!accountToNameMap || !accountToNameMap[g1]) {
+                            Log.alert('[ExpensiMark] Missing account name', {accountID: g1});
+                            return '@Hidden';
+                        }
 
-                    return `@${extras.accountIdToName[g1]}`;
+                        return `@${accountToNameMap[g1]}`;
+                    }
+                    return Str.removeSMSDomain(g2)
```

Comment:
What's the purpose of these changes?

Reply:
+1

Reply:
The failling tests are because I didn't changes the fields in the test files yet, I am sorry.

For  `accountToNameMap`, I am following the pattern in the file for `reportToNameMap`:

https://github.com/Expensify/expensify-common/blob/321e4a7f5e618000be812d5b7dc76d00f81084f9/lib/ExpensiMark.js#L545

`accountToNameMap` is just a variable that store `extras.accountIdToName`. It is the same thing.
Should I revert it?